simple_history/models.py

```diff
@@ -668,7 +668,7 @@ def create_historical_record_m2ms(self, history_instance, instance):
 
             insert_rows = []
 
-            through_field_name = type(original_instance).__name__.lower()
+            through_field_name = field.m2m_field_name()
```

Comment:
Can't find anything on this method in the docs, so it could be useful adding a comment like this:
```suggestion
            # `m2m_field_name()` is part of Django's internal API
            through_field_name = field.m2m_field_name()
```